app/src/main/java/xyz/jhughes/laundry/fragments/MachineFragment.java

```diff
@@ -319,4 +322,29 @@ public void postSnackbar(String status, int length) {
         Snackbar snackbar = Snackbar.make(rootView, status, length);
         snackbar.show();
     }
+
+    public void onLock(){
```

Comment:
Since this orientation code is really specific can we make it some sort of static "OrientationUtils" class with this as a static method that takes an Activity. Also if you pulled this from somewhere specific can you add a link to that? 

I'm worried we will forget what this actually does down the line.